tsdx.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
         ],
         inject: false,
         // only write out CSS for the first bundle (avoids pointless extra files):
-        extract: !!options.writeMeta && 'dist/main.css',
+        extract: !!options.writeMeta && 'main.css',
```

Comment:
`'dist/main.css'` --> `'main.css'`
Not sure what is the reason for the change as I can't see it from the PR description clearly. Should this be explained?

Reply:
The output remains as it was. I assume it's about either postcss plugin or tsdx update changing the extract path to be relative to the output path not the config path. No idea what the reason for this is, nor is it explained in the plugin [documentation](https://www.npmjs.com/package/rollup-plugin-postcss). TSDX is notoriously baldy documented as what comes to customization, so no explanation there eiter.

Reply:
And now it's documented 😅